### PR TITLE
Fix: check  if enabling operation will uninstall controller from some clusters

### LINKED ISSUE #5138: [Bug] Specifying a cluster to enable fluxcd will delete the fluxcd installed on the previously joined cluster
**Describe the bug**

Specifying a cluster to enable fluxcd will delete the fluxcd installed on the previously joined cluster
image
image
image

**To Reproduce**

**Expected behavior**

We hope that the installation of fluxcd in the specified cluster will not affect the installed fluxcd of the previously

**Screenshots**

**KubeVela Version**
1.6.2

**Cluster information**
1.21.5

**Additional context**

### PR DESCRIPTION
**Pull request by wangyikewxgm**
check an enabling operation whether uninstall addon from clusters

image

### Description of your changes

Fixes #5138 

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/kubevela/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: references/cli/addon.go [hunk 1]
```diff
@@ -217,6 +224,7 @@ func NewAddonEnableCommand(c common.Args, ioStream cmdutil.IOStreams) *cobra.Com
 	cmd.Flags().BoolVarP(&skipValidate, "skip-version-validating", "s", false, "skip validating system version requirement")
 	cmd.Flags().BoolVarP(&overrideDefs, "override-definitions", "", false, "override existing definitions if conflict with those contained in this addon")
 	cmd.Flags().BoolVarP(&dryRun, FlagDryRun, "", false, "render all yaml files out without real execute it")
+	cmd.Flags().BoolVarP(&skipCheckUninstallFromClusters, "skip-check-uninstall-clusters", "", false, "skip check 'clusters' parameters whether uninstall addon from clusters")
```
Comment (wonderflow) [inline references/cli/addon.go:227]:
```suggestion
	cmd.Flags().BoolVarP(&yes2all, "yes", "y", false, "all checks will be skipped and the default answer is yes for all validation check.")
```

Please merge with the `skip-version-validating` option.

We should avoid long and boring arguments.

File: references/cli/addon.go [hunk 2]
```diff
@@ -217,6 +224,7 @@ func NewAddonEnableCommand(c common.Args, ioStream cmdutil.IOStreams) *cobra.Com
 	cmd.Flags().BoolVarP(&skipValidate, "skip-version-validating", "s", false, "skip validating system version requirement")
 	cmd.Flags().BoolVarP(&overrideDefs, "override-definitions", "", false, "override existing definitions if conflict with those contained in this addon")
 	cmd.Flags().BoolVarP(&dryRun, FlagDryRun, "", false, "render all yaml files out without real execute it")
+	cmd.Flags().BoolVarP(&yes2all, "yes", "y", false, "all checks will be skipped and the default answer is yes for all validation check.")
```
Comment (wonderflow) [inline references/cli/addon.go:227]: 其他这种选择也得囊括进来。

File: references/cli/addon.go [hunk 3]
```diff
@@ -1201,3 +1209,51 @@ func splitSpecifyRegistry(name string) (string, string, error) {
...
+	var disableClusters, installedClusters []string
+	for c := range status.Clusters {
+		if _, ok := clusters[c]; !ok {
+			disableClusters = append(disableClusters, c)
+		}
+		installedClusters = append(installedClusters, c)
+	}
+	fmt.Println(color.New(color.FgRed).Sprintf("Previously, the addon was installed on clusters %s, but this operation will disable the addon from clusters: %s \n", generateClustersInfo(installedClusters), generateClustersInfo(disableClusters)))
+	input := NewUserInput()
+	if !input.AskBool("Do you want continue?", &UserInputOptions{AssumeYes: false}) {
```
Comment (wonderflow) [inline references/cli/addon.go:1243]:
```suggestion
	if !input.AskBool("Do you want to continue?", &UserInputOptions{AssumeYes: false}) {
```

File: references/cli/addon.go [hunk 4]
```diff
@@ -197,7 +200,11 @@ func NewAddonEnableCommand(c common.Args, ioStream cmdutil.IOStreams) *cobra.Com
 				if filepath.IsAbs(addonOrDir) || strings.HasPrefix(addonOrDir, ".") || strings.HasSuffix(addonOrDir, "/") {
 					return fmt.Errorf("addon directory %s not found in local", addonOrDir)
```
Comment (wonderflow) [inline references/cli/addon.go:201]:
```suggestion
					return fmt.Errorf("addon directory %s not found in local file system", addonOrDir)
```

File: references/cli/addon.go [hunk 5]
```diff
@@ -1201,3 +1209,51 @@ func splitSpecifyRegistry(name string) (string, string, error) {
 		return "", "", fmt.Errorf("invalid addon name, you should specify name only  <addonName>  or with registry as prefix <registryName>/<addonName>")
 	}
 }
+
+func checkUninstallFromClusters(ctx context.Context, k8sClient client.Client, addonName string, args map[string]interface{}) error {
+	status, err := pkgaddon.GetAddonStatus(ctx, k8sClient, addonName)
+	if err != nil {
+		return fmt.Errorf("failed to check addon is enabled err: %w", err)
```
Comment (wonderflow) [inline references/cli/addon.go:1216]:
```suggestion
		return fmt.Errorf("failed to check addon status: %w", err)
```

**Review by Somefive — APPROVED** [review]

**Pull request merged**